_i18n/en/documentation/general/other-platforms.md

```diff
@@ -0,0 +1,12 @@
+However attractive the title may sound, AntennaPod is not and will not be available on Windows, iOS or the Linux desktop. The app is made by a small team of volunteers, and making the app for these other platforms would be a lot of work.
+
+There are (open source) alternatives, however, which we would like to point you to:
+App/Application|Windows|iOS|Linux
+---|---|---|---
+[Kasts](https://apps.kde.org/kasts/)|No|No|Yes
```

Comment:
Yes/no tables are quite hard to read at a glance IMO. How about leaving it empty for "no" and adding a check mark symbol (✓) for "yes"? Then we can also add do-not-translate around the table

Reply:
I've added a cross for 'no' because with all the empty spaces the table became hard to read. If we have better table formatting in future, we can remove the crosses.

Reply:
hmm, but with crosses, we have the same problem as before - completely filled and hard to see at a glance which is which

Reply:
> How about centering the symbols below the table heading?

That'd be nice indeed. But that's also [table styling](https://www.codeply.com/p/O8BLz8ic1X) that can't be done in markdown. So I'll save that for later.

Reply:
Markdown does support centering table content

Reply:
Sorry, I searched but apparently not properly. Will update it.